### ISSUE TITLE
fix permissions for /repos/environments

### ISSUE THREAD

**Issue by cdeboever3** [frazer-lab/cluster#48]
I'm unable to commit changes to this repo. I think it needs to be configured as a shared repo (which I did just now), but I still need the permissions fixed and I think this will require root.

http://stackoverflow.com/questions/6448242/git-push-error-insufficient-permission-for-adding-an-object-to-repository-datab

```
(cardips)[/r/environments]fl-hn1 $ git commit 
error: insufficient permission for adding an object to repository database .git/objects
error: insufficient permission for adding an object to repository database .git/objects
error: Error building trees
```

**Comment by tatarsky** [comment]
OK. I suspect if i chown the entire repos tree to you it will work OK right? Sorry if I'm being dense I'm running between two locations ;)

**Comment by cdeboever3** [comment]
I think that will fix it.

**Comment by tatarsky** [comment]
Give it a try!

**Comment by cdeboever3** [comment]
Well I think that worked, but somehow the remotes for the repo were changed
to the cardips-pipeline repo we have (I'm guessing a user error unrelated
to this issue), and that made for a bad time when I tried to push changes.
So I ended up just re-cloning the environments repo and moving the old one
from /repos/environments to /repos/old/environments. The only changes were
ones I made and a change to the speedseq module (adding pysam) that I
copied into the new version. I'll delete /repos/old/environments in a few
days when I know I didn't mess something up. ugh
